crud/select/compat/common.lua

```diff
@@ -1,6 +1,20 @@
+local ratelimit = require'crud.ratelimit'
```

Comment:
Nit: Please, use `local bar = require('foo.bar')` form (with parentheses). See https://www.tarantool.io/en/doc/latest/dev_guide/lua_style_guide/#idioms-and-patterns.

Reply:
Thanks, I'll remember that.